### ISSUE TITLE
Fast Paas failing : Ensures ARIA attributes are allowed for an element's role in fabric multi select dropdown.

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#15888]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^7.126.0",
- **Browser and OS versions**: Chrome and Windows 10

### Describe the issue:
Fast Paas is failing with error "Ensures ARIA attributes are allowed for an element's role" in fabric multi select dropdown when any item selected.

#### Actual behavior:
Fast Paas is failing with error "Ensures ARIA attributes are allowed for an element's role" in fabric multi select dropdown when any item selected.

#### Expected behavior:
Fast Paas should pass when any item selected in multi select dropdown control.
<img width="960" alt="multiselect1" src="https://user-images.githubusercontent.com/48669559/98636263-84f3f800-234c-11eb-8182-bb6e20f22f67.png">
<img width="893" alt="multiselect2" src="https://user-images.githubusercontent.com/48669559/98636271-89201580-234c-11eb-9337-23cc89759503.PNG">

**Comment by paulgildea** [comment]
@Sandhya-aluri Thanks for filing this issue.
